code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -331,3 +331,20 @@
 	icon_dead = "penny_dead"
 	icon_rest = "penny_rest"
 	holder_type = /obj/item/holder/cat/penny
+/mob/living/simple_animal/cat/crusher // Goodbye Runtime, make way for the new girl in town.
+	name = "Crusher"
+	desc = "Crusher, nicknamed Dr. Crusher, is a young, smart and cuddly cat. She never lets an opportunity pass to receive some pets or prey on some unsuspecting mice."
+	named = TRUE
+	gender = FEMALE
+	icon_state = "styx"
+	icon_state = "styx"
+	icon_living = "styx"
+	icon_dead = "styx_dead"
+	icon_rest = "styx_rest"
+	can_nap = 1
+	holder_type = /obj/item/holder/cat/crusher
+
+/mob/living/simple_animal/cat/crusher/examine(mob/user)
+	..()
+	if(stat == DEAD)
+		to_chat(user, "Crusher's dead. How could this have happened? She counted on you!")
```

Comment:
is this necessary?

Reply:
Yes. I dont know what you mean here. Every pet has this, except Ives I think. Stays.